pytorch_lightning/loggers/neptune.py

```diff
@@ -293,9 +293,10 @@ def __init__(
     def _retrieve_run_data(self):
         try:
             self._run_instance.wait()
-            self._run_short_id = self.run._short_id  # skipcq: PYL-W0212
+            self._run_short_id = self._run_instance["sys/id"].fetch()
```

Comment:
Does this need a deprecation warning first before the breaking change is made?

Reply:
There is no breaking change. `_short_id` is internal property of our `Run` (which exists in the context of `neptune-client` library and `neptune` package). `["sys/id"].fetch()` is just a gentle way of access same value. There is no change from the user perspective.

Reply:
Or I'm missing something :wink: